web/src/main/java/org/akaza/openclinica/control/managestudy/ViewStudySubjectServlet.java

```diff
@@ -309,18 +312,16 @@ public void processRequest() throws Exception {
                     }
```

Comment:
Original formatting was ok.  It was the diff tool wrapping the text.  Cleaned up some other formatting in method though included unneeded carriage returns and unnecessary comments.